detection-rules/attachment_adobe_image_lure_fts.yml

```diff
@@ -0,0 +1,32 @@
+name: "Attachment: Adobe image lure with with suspicious link from first time sender"
+description: "Detects Adobe phishing messages with an Adobe logo attached, with suspicious link language
+  from a first-time sender."
+type: "rule"
+severity: "medium"
+source: |
+  type.inbound
+  and length(filter(attachments, .file_type not in~ ('png', 'jpeg', 'jpg', 'bmp'))) == 0
+  and length(body.links) >0
+  and all(body.links, .display_text is null)
```

Comment:
this is surprising. would have thought the phrases below would be on the display text

Reply:
The image is hyperlinked so no need for display text